aks/redis/variables.tf

```diff
@@ -71,13 +72,15 @@ variable "azure_sku_name" {
 }
 
 variable "azure_minimum_tls_version" {
-  type    = string
-  default = "1.2"
+  type     = string
+  nullable = true
+  default  = "1.2"
 }
 
 variable "azure_public_network_access_enabled" {
-  type    = bool
-  default = false
+  type     = bool
+  nullable = true
```

Comment:
I don't think booleans should be `nullable` since `false` acts the same as `null`.

Reply:
I had updated to nullable = false. 
See this https://github.com/hashicorp/terraform/pull/29832#issue-1039803860, for reference